tests/testServer.js

```diff
@@ -1,6 +1,6 @@
-const app = require('./testApp.js')(app);
+const app = require('./testApp.js');
```

Comment:
Not sure if you changed the file name from ./testApp.js to ./testIndex.js. If not, disregard this comment as it's showing two different things on my end.